src/test/snapshot-controller/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+
+appVersion: 2.1.1
+name: snapshot-controller
+description: A Helm chart for snapshot-controller
+version: 2.1.1
```

Comment:
`version` is not `appVersion`, and we usually start with something like `0.1.0`, in which `0` means it's unstable. When the chart becomes stable enough, we can promote it to `1.0.0`.

Reply:
OK